### PR TITLE
Added another exclude example

### PR DESCRIPTION
**Pull request by royduin**

### CODE REVIEW

File: docs/src/indexing-flow.md [hunk 1]
```diff
@@ -79,6 +79,32 @@ add_filter( 'algolia_should_index_user', 'filter_user', 10, 2 );
...
+ *
+ * @return bool
+ */
+function filter_post( bool $should_index, WP_Post $post )
+{
+    if ( false === $should_index ) {
+        return false;
+    }
+
+    return get_post_meta($post->ID, '_yoast_wpseo_meta-robots-noindex', true) == 1 ? false : true;
```
Comment (rayrutjes) [inline docs/src/indexing-flow.md]: Is there a specific reason to not use strict equality check here?

File: docs/src/indexing-flow.md [hunk 2]
```diff
@@ -79,6 +79,32 @@ add_filter( 'algolia_should_index_user', 'filter_user', 10, 2 );
 In the above example, User with ID 1 would never get indexed. This example is trivial, but at least gives you a quick overview on how to filter your items to index.
 Also note how we return `false` early on if the decision has already been taken to not index the content.
 
+**Another example to exclude posts from the searchable_posts index where the noindex option from the Yoast SEO plugin is set to "noindex"**
+```php
```
Comment (rayrutjes) [inline docs/src/indexing-flow.md]: Do you mind adding a small warning here to not confuse user not using the YOAST plugin?

```
<div class="alert alert-warning">The following plugin assumes you are using the YOAST SEO plugin for WordPress</div>
```

**Comment by rayrutjes** [comment]
Thanks a lot @royduin, valuable example! I wish I had time to write a bunch more ;)
Just suggested 2 small improvements, I'd gladly merge it afterwards!

File: docs/src/indexing-flow.md (hunk 1 above)
Comment (royduin) [inline docs/src/indexing-flow.md]: Yes:
```
0 = Use default
1 = noindex
2 = index
```

File: docs/src/indexing-flow.md (hunk 2 above)
Comment (royduin) [inline docs/src/indexing-flow.md]: It's already in the bold text, that's not enough?

**Comment by royduin** [comment]
You're welcome, I needed this so I though let's share it with the world ;)

File: docs/src/indexing-flow.md (hunk 1 above)
Comment (rayrutjes) [inline docs/src/indexing-flow.md]: It looks like you will get a string back indeed.

File: docs/src/indexing-flow.md (hunk 2 above)
Comment (rayrutjes) [inline docs/src/indexing-flow.md]: No sorry, I'd rather make it crystal clear with some specific formatting. Do you mind adding the suggested alert?

Comment (royduin) [inline docs/src/indexing-flow.md]: Alright, I've created another commit with the notice.

**Comment by rayrutjes** [comment]
Thanks a lot @royduin !

I will deploy the updated doc later today.

**Comment by royduin** [comment]
Nice! Thanks for merging!

**Pull request merged**